pkgs/applications/misc/swaysettings/default.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, fetchFromGitHub
+, accountsservice
+, appstream-glib
+, dbus
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gsettings-desktop-schemas
+, gtk3
+, json-glib
+, libgee
+, libhandy
+, libxml2
+, meson
+, ninja
+, pantheon
+, pkg-config
+, python3Full
+, stdenv
+, vala
+, wrapGAppsHook
+}:
+stdenv.mkDerivation rec {
+  pname = "swaysettings";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwaySettings";
+    rev = "v0.3.0";
+    hash = "sha256-2bbB+37t6chbdnOSpIolAsy7aD9i1UizWqkcF8Frfsk=";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    gettext
+    meson
+    ninja
+    pkg-config
+    python3Full
```

```suggestion
    python3
```

The full Python is for interactive use primarily.

---

corrected